source/agora/node/BlockStorage.d

```diff
@@ -809,22 +808,6 @@ public class BlockStorage : IBlockStorage
         }
     }
 
-    /***************************************************************************
-
-        Remove the index file.
-
-        Params:
-            path = path to the data directory
-
-    ***************************************************************************/
-
-    public static void removeIndexFile (string path)
```

Comment:
Was this function unused? It seems out of place in this commit.

Reply:
Yep it was. And as I was grepping for `buildPath` I came accross this. Want it in another commit ?

Reply:
I guess it doesn't matter much if it was no-op.

Reply:
It was actually used in the integration tests. So I reverted the `rm`.